### PR TITLE
Auditv2 fix unzip redux

### PR DESCRIPTION
**Pull request by escherize**
cherry picked from #34146

### CODE REVIEW

File: src/metabase/api/collection.clj [hunk 1]
```diff
@@ -81,7 +81,7 @@
                                (collection/visible-collection-ids->honeysql-filter-clause
                                 :id
                                 (collection/permissions-set->visible-collection-ids @api/*current-user-permissions-set*))]
-                    :order-by [[:type :nulls-first] [:%lower.name :asc]]}) collections
+                    :order-by [[:type :asc] [:%lower.name :asc]]}) collections
```
Comment (noahmoss) [inline src/metabase/api/collection.clj:84]: what does this change do?

**Review by noahmoss — APPROVED** [review]

File: src/metabase/api/collection.clj (hunk 1 above)
Comment (escherize) [reply src/metabase/api/collection.clj:84]: `:nulls-first` breaks this query on mysql. This preserves the proper ordering (instance_analytics should be last) and actually runs on mysql.

**Comment by iethree** [comment]
be careful how you use `redux` - some of us find it triggering.

**Comment by escherize** [comment]
Oh true: I dont't want the mob eXamining that message and over reacting 😂

**Pull request merged**